src/main/java/fi/metatavu/kuntaapi/server/index/IndexRemoveNewsArticle.java

```diff
@@ -0,0 +1,25 @@
+package fi.metatavu.kuntaapi.server.index;
+
+public class IndexRemoveNewsArticle implements IndexRemove {
+
+  private String newsArticleId;
+
+  @Override
+  public String getId() {
+    return newsArticleId;
+  }
+
+  @Override
+  public String getType() {
+    return "newsarticle";
+  }
+  
+  public String getNewsArticleId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getId" on line 8. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)